plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -186,28 +186,6 @@ public static QueryRunner createS3DeltaLakeQueryRunner(
                 additionalSetup);
     }
 
-    public static QueryRunner createAbfsDeltaLakeQueryRunner(
-            String catalogName,
-            String schemaName,
-            Map<String, String> extraProperties,
-            Map<String, String> connectorProperties,
-            HiveHadoop testingHadoop)
-            throws Exception
-    {
-        return createDockerizedDeltaLakeQueryRunner(
-                catalogName,
-                schemaName,
-                ImmutableMap.of(),
-                extraProperties,
-                ImmutableMap.<String, String>builder()
-                        .put("hive.azure.abfs-storage-account", requiredNonEmptySystemProperty("testing.azure-abfs-account"))
```

Comment:
`requiredNonEmptySystemProperty` seems to be unused now.

Reply:
it is still used in a different class. this should be a helper (not specific to delta), but this as a separate PR